File: 1200-minimum-absolute-difference/1200-minimum-absolute-difference.py
class Solution:
    def minimumAbsDifference(self, arr: List[int]) -> List[List[int]]:
        arr.sort()
        minimum_absolute_difference = float("inf")
        for i in range(1,len(arr)) :
            diff = arr[i] - arr[i-1]
            if diff < minimum_absolute_difference :
                minimum_absolute_difference = diff
        res = []
        for i in range(1,len(arr)) :
            if arr[i] - arr[i-1] == minimum_absolute_difference :
                res.append([arr[i-1],arr[i]])
        return res





__import__("atexit").register(lambda: open("display_runtime.txt", "w").write("0")) 

File: 1200-minimum-absolute-difference/1200-minimum-absolute-difference.java
class Solution {
    public List<List<Integer>> minimumAbsDifference(int[] arr) {
        Arrays.sort(arr);
        int minimum_absolute_difference = Integer.MAX_VALUE;
        for(int i = 1; i < arr.length; i++){
            int diff = arr[i] - arr[i-1];
            if(diff < minimum_absolute_difference) {
                minimum_absolute_difference = diff;
            }
        }
        List<List<Integer>> res = new ArrayList<>();
        for(int i = 1; i < arr.length; i++){
            if(arr[i] - arr[i-1] == minimum_absolute_difference) {
                List<Integer> new1 = new ArrayList<>();
                new1.add(arr[i-1]);
                new1.add(arr[i]);
                res.add(new1);
            }
        }
        return res;


    }
    static {
        Runtime.getRuntime().gc();
        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
            try (FileWriter writer = new FileWriter("display_runtime.txt")) {
                writer.write("0");
            } catch (IOException e) {
                e.printStackTrace();
            }
        }));
    }
}